prisma/schema.prisma

```diff
@@ -148,6 +148,7 @@ model PublicCloudRequestedProject {
   created                  DateTime            @default(now())
   accountCoding            String
   budget                   Budget
+  billingGroup             String
```

Comment:
I don't believe that we have a billing group

Reply:
Actually, we changed billing group to account coding, so we would need to remove billing group from the seed file and replace it with account coding